webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/RegistrationCommandsPageSelectors.js

```diff
@@ -9,11 +9,24 @@ import {
   selectAPIResponse,
 } from '../../../redux/API/APISelectors';
 
+import { STATUS } from '../../../constants';
+
 // Form API Data
 
 export const selectAPIStatusData = state =>
   selectAPIStatus(state, REGISTRATION_COMMANDS_DATA);
 
+export const selectApiDataErrorCode = state => {
+  const isError =
+    selectAPIStatus(state, REGISTRATION_COMMANDS_DATA) === STATUS.ERROR;
```

Comment:
I think `selectAPIStatus` only accepts a single argument
```suggestion
    selectAPIStatus(state) === STATUS.ERROR;
```

Reply:
Nope, it needs to know which api status key it needs to find, the state has all api calls/urls

Reply:
Oh I see, I misread it as reusing `selectAPIStatusData` from above. So this is what I thought would be used.
```suggestion
    selectAPIStatusData(state) === STATUS.ERROR;
```